R/ci.R

```diff
@@ -111,3 +111,39 @@ appveyor_info <- function(base_path = proj_get()) {
 
   list(url = url, img = img)
 }
+
+#' @section `use_gitlab_ci()`:
+#' Adds a basic `.gitlab-ci.yml` to the top-level directory of a package. This is
+#' a configuration file for the [Gitlab CI/CD](https://docs.gitlab.com/ee/ci/) continuous
+#' integration service for Gitlab.
+#' @export
+#' @rdname ci
+use_gitlab_ci <- function() {
+  check_uses_git()
+  new <- use_template(
+    "gitlab-ci.yml",
+    ".gitlab-ci.yml",
+    ignore = TRUE
+  )
+  if (!new) return(invisible(FALSE))
+
+  invisible(TRUE)
```

Comment:
I think all the return bits can be simplified to `invisible(new)`.

Reply:
Updated in https://github.com/r-lib/usethis/pull/562/commits/4fe65038e08dab007556adb5602e5760926ad925